src/site/apt/development/performing-a-release.apt

```diff
@@ -152,37 +152,50 @@ mvn docck:check
+|    (3) DSA (sign only)\                             |
+|    (4) RSA (sign only)\                             |
+|    (7) DSA (set your own capabilities)\             |
+|    (8) RSA (set your own capabilities)\             |
+|    (9) ECC and ECC\                                 |
+|   (10) ECC (sign only)\                             |
+|   (11) ECC (set your own capabilities)\             |
+|   (13) Existing key\                                |
+|   (14) Existing key from card                       | 9
 *-----------------------------------------------------+-------------+
-| Please specify how long the key should be valid.    | 
-|    0 = key does not expire                          |
-| <n>  = key expires in n days                        |
-| <n>w = key expires in n weeks                       |
-| <n>m = key expires in n months                      |
-| <n>y = key expires in n years                       |
-| Key is valid for? (0)                               | 0
+| Please select which elliptic curve you want:\       |
+|    (1) Curve 25519\                                 |
+|    (3) NIST P-256\                                  |
+|    (4) NIST P-384\                                  |
+|    (5) NIST P-521\                                  |
+|    (6) Brainpool P-256\                             |
+|    (7) Brainpool P-384\                             |
+|    (8) Brainpool P-512\                             |
+|    (9) secp256k1                                    | 1
 *-----------------------------------------------------+-------------+
-| Key does not expire at all                          |
-| Is this correct? (y/N)                              | y
+| Please specify how long the key should be valid.\   |
+|          0 = key does not expire\                   |
+|       <n>  = key expires in n days\                 |
+|       <n>w = key expires in n weeks\                |
+|       <n>m = key expires in n months\               |
+|       <n>y = key expires in n years                 | 4y
 *-----------------------------------------------------+-------------+
 | Real name:                                          | <your real name>
 *-----------------------------------------------------+-------------+
 | Email address:                                      | <your email address>
 *-----------------------------------------------------+-------------+
+| Comment                                             | <empty>
+*-----------------------------------------------------+-------------+
 | Change (N)ame, (C)omment, (E)mail or (O)kay/(Q)uit? | O
 *-----------------------------------------------------+-------------+
 
-  
   * Execute <<<gpg --list-sigs ">>><your real name><<<" && gpg --armor --export ">>><your real name><<<">>> and
-  visit {{http://pgp.mit.edu/}} to distribute your public key, so users can verify your files. 
+  visit {{https://keyserver.ubuntu.com/}} to distribute your public key, so users can verify your files.
```

Comment:
Why only this server ... https://hkps.pool.sks-keyservers.net/ is also popular in maven world